app/controllers/api/v2/template_combinations_controller.rb

```diff
@@ -36,9 +62,20 @@ def destroy
       end
 
       def find_parent_config_template
-        @config_template = ConfigTemplate.authorized(:view_templates).find(params[:config_template_id])
-        not_found unless @config_template
-        @config_template
+        @config_template = ConfigTemplate.authorized(:view_templates).find(params[:config_template_id]) rescue nil
+      end
+
+      def requested_resource
```

Comment:
Sorry, I don't think I was clear by putting this comment on the wrong line.  I meant to ask whether `find_parent_config_template` is required, because config_template_id is in the allowed_nested_id list.  Then everything is handled via the nested object lookup in the base controller, and all we need `nested_obj`.

e.g. https://github.com/domcleal/foreman/commit/a298ae5b15c81846cb2dffe034845d102fa17bb9

Tests pass for me, but to be honest, they're not very thorough...